api/api.go

```diff
@@ -37,6 +39,43 @@ type SubmissionInfo struct {
 	State string `json:"state"`
 }
 
+// StatusInfo contains information about a user's status on a particular language track.
+type StatusInfo struct {
+	Language string `json:"track_id"`
+	Recent   struct {
+		Problem   string `json:"problem"`
+		Submitted string `json:"submitted_at"`
+	}
+	Fetched []string `json:"fetched"`
+	Skipped []string `json:"skipped"`
```

Comment:
Lately I've been trying to name collections as the name of what's in them, in the plural. That might not be a good choice here, because we'll end up with two long names where the suffix is repeated. I'm open to suggestions, though.